apps/dotcom/client/src/pages/admin.tsx

```diff
@@ -1,17 +1,96 @@
+import { ZStoreData } from '@tldraw/dotcom-shared'
 import { useCallback, useEffect, useRef, useState } from 'react'
 import { Navigate } from 'react-router-dom'
 import { fetch } from 'tldraw'
+import { TlaButton } from '../tla/components/TlaButton/TlaButton'
 import { useTldrawUser } from '../tla/hooks/useUser'
+import styles from './admin.module.css'
+
+// Helper component for structured data display
+function StructuredDataDisplay({ data }: { data: ZStoreData }) {
+	const [copied, setCopied] = useState(false)
+
+	const handleCopy = async () => {
+		try {
+			await navigator.clipboard.writeText(JSON.stringify(data, null, 2))
+			setCopied(true)
+			setTimeout(() => setCopied(false), 2000)
+		} catch (err) {
+			console.error('Failed to copy:', err)
+		}
+	}
+
+	return (
+		<div className={styles.structuredData}>
+			<TlaButton onClick={handleCopy} variant="secondary" className={styles.copyButton}>
+				{copied ? 'Copied!' : 'Copy JSON'}
+			</TlaButton>
+			<pre className={styles.dataDisplay}>{JSON.stringify(data, null, 2)}</pre>
+		</div>
+	)
+}
+
+// Helper component for user data summary
+function UserDataSummary({ userData }: { userData: any }) {
+	const getUserInfo = () => {
+		const user = userData?.USER?.[0] || userData?.user || userData
+		const files = userData?.FILE || []
+		const deletedFiles = files.filter((f: any) => f.deletedAt)
+		const activeFiles = files.filter((f: any) => !f.deletedAt)
```

Comment:
This doesn't work, we probably need to look at `isDeleted` field? Also not sure if we are correctly getting the `files` or maybe the `user` in the first place.